src/XCTrace/XCBacktrace.class.st

```diff
@@ -16,6 +16,12 @@ XCBacktrace class >> sample: aXCTraceSample node: aXMLElement [
 		yourself
 ]
 
+{ #category : #accessing }
+XCBacktrace >> frameNameAt: pos [
+	
+	^ (self frames at: pos) attributeAt: 'name'
```

Comment:
I propose we refactor this a bit differently in the next iteration.

```suggestion
	^ (self frameAt: pos) name
```

Otherwise we have a dependency to xml here